stage2/README.md

```diff
@@ -1,100 +1,101 @@
-# RS School. Курс JS/Frontend-разработка. Stage#2
-
-- Для зачисления необходимо успешно выполнить задания первого этапа, пройти собеседование с ментором
-- Цели второго этапа обучения:
-  - Получение студентами знаний и опыта работы с TypeScript
-  - Получение студентами знаний и опыта разработки UI компонентов
-  - Подготовка студентов к прохождению технических интервью
-  - Подтоговка студентов к выступлению на английском языке
-  - Выполнение студентами курсового проекта
-- Время обучения: 12 недель
-- Формат: Cамообучение + вебинары + общение в Discord. Практические занятия проверяются и оцениваются ментором и во время кросс-чек. В ходе обучения проводятся тренировочные интервью у разных менторов.
-
-# Программа обучения
-
-## Неделя #1. 22/11/2022
-- [Модуль: "JavaScript. Classes & Prototypes"](modules/classes-prototypes/)
-- [Модуль: Object-Oriented Programming Basics](modules/oop-basics/)
-- [Задачи "Codewars-OOP"](../tasks/codewars/Codewars-2022Q3-OOP.md) 
-- [Кросс-чек проверка](https://docs.rs.school/#/cross-check-flow) проекта ["Self-introduction"](modules/self-introduction/)
-- [Кросс-чек проверка](https://docs.rs.school/#/cross-check-flow) проекта ["songbird"](../tasks/songbird/songbird-2022q3.md)
+# RS School. JS/Frontend Development Course. Stage#2
+
+- To enroll, students must successfully complete the tasks of the first stage and pass an interview with a mentor
+- Goals of the second stage of training:
+  - Students acquire knowledge and experience working with TypeScript
+  - Students acquire knowledge and experience in developing UI components
+  - Preparing students for technical interviews
+  - Preparing students for presentations in English
+  - Students complete a team project
+- Training time: 14 weeks
+- Format: Self-study + webinars + communication on Discord. Practical lessons are checked and evaluated by a mentor and during cross-checks. Training interviews are held with different mentors during the course.
+
+# Training Program
+
+## Week #1
+- [Module: "JavaScript. Classes & Prototypes"](modules/classes-prototypes/)
+- [Module: Object-Oriented Programming Basics](modules/oop-basics/)
+- [Codewars-OOP Tasks](../tasks/codewars/Codewars-2022Q3-OOP.md) 
+- [Cross-check](https://docs.rs.school/#/cross-check-flow) of the ["Self-introduction" project](modules/self-introduction/)
+- [Cross-check](https://docs.rs.school/#/cross-check-flow) of the ["songbird" project](../tasks/songbird/songbird-2022q3.md)
 - https://github.com/mikhama/core-js-101
 
 <details>
-  <summary markdown="span">Активности менторов</summary>
+  <summary markdown="span">Mentor activities</summary>
 
-- [Подтверждение регистрации на курс](https://docs.rs.school/#/mentoring-kick-off)
+- [Confirming course registration](https://docs.rs.school/#/mentoring-Kick-off)
```

Comment:
Unfortunately, our documentation is case-sensitive. Please fix the link https://docs.rs.school/#/mentoring-**kick**-off